<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 6.1.1 (Python 3.12.1 on win32)" generated="20240109 23:49:16.937" rpa="false" schemaversion="4">
<suite id="s1" name="Sanbercode-Quiz-4-Bootcamp-QA-batch-52" source="D:\Sanbercode\Sanbercode-Quiz-4-Bootcamp-QA-batch-52">
<suite id="s1-s1" name="Suites" source="D:\Sanbercode\Sanbercode-Quiz-4-Bootcamp-QA-batch-52\suites">
<suite id="s1-s1-s1" name="bookSuite" source="D:\Sanbercode\Sanbercode-Quiz-4-Bootcamp-QA-batch-52\suites\bookSuite.robot">
<test id="s1-s1-s1-t1" name="User should be able to book flight" line="18">
<kw name="Run Keywords" library="BuiltIn" type="SETUP">
<arg>Open Flight Application</arg>
<arg>AND Login with Valid Credentials</arg>
<doc>Executes all the given keywords in a sequence.</doc>
<kw name="Open Flight Application" library="base">
<kw name="Open Application" library="AppiumLibrary">
<arg>remote_url=${REMOTE_URL}</arg>
<arg>platformName=${PLATFORM_NAME}</arg>
<arg>platformVersion=${PLATFORM_VERSION}</arg>
<arg>deviceName=${DEVICE_NAME}</arg>
<arg>appPackage=${APP_PACKAGE}</arg>
<arg>appActivity=${APP_ACTIVITY}</arg>
<doc>Opens a new application to given Appium server.
Capabilities of appium server, Android and iOS,
Please check https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/server-args.md
| *Option*            | *Man.* | *Description*     |
| remote_url          | Yes    | Appium server url |
| alias               | no     | alias             |
| strict_ssl          | No     | allows you to send commands to an invalid certificate host like a self-signed one. |</doc>
<kw name="Capture Page Screenshot" library="AppiumLibrary">
<doc>Takes a screenshot of the current page and embeds it into the log.</doc>
<msg timestamp="20240109 23:49:31.614" level="FAIL">No application is open</msg>
<status status="FAIL" starttime="20240109 23:49:31.613" endtime="20240109 23:49:31.625"/>
</kw>
<msg timestamp="20240109 23:49:31.629" level="WARN">Keyword 'Capture Page Screenshot' could not be run on failure: No application is open</msg>
<msg timestamp="20240109 23:49:31.630" level="FAIL">WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to the remote server. Original error: socket hang up
Stacktrace:
UnknownError: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to the remote server. Original error: socket hang up
    at UIA2Proxy.command (C:\Users\ASUS\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\jsonwp-proxy\proxy.js:274:13)
    at runMicrotasks (&lt;anonymous&gt;)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at AndroidUiautomator2Driver.commands.getDevicePixelRatio (C:\Users\ASUS\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\node_modules\appium-uiautomator2-driver\lib\commands\viewport.js:14:10)
    at AndroidUiautomator2Driver.fillDeviceDetails (C:\Users\ASUS\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\node_modules\appium-uiautomator2-driver\lib\driver.js:244:28)
    at AndroidUiautomator2Driver.createSession (C:\Users\ASUS\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\node_modules\appium-uiautomator2-driver\lib\driver.js:230:7)
    at AppiumDriver.createSession (C:\Users\ASUS\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\lib\appium.js:387:35)</msg>
<status status="FAIL" starttime="20240109 23:49:17.608" endtime="20240109 23:49:31.632"/>
</kw>
<status status="FAIL" starttime="20240109 23:49:17.608" endtime="20240109 23:49:31.632"/>
</kw>
<status status="FAIL" starttime="20240109 23:49:17.606" endtime="20240109 23:49:31.633"/>
</kw>
<status status="FAIL" starttime="20240109 23:49:17.605" endtime="20240109 23:49:31.633">Setup failed:
WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to the remote server. Original error: socket hang up
Stacktrace:
UnknownError: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to the remote server. Original error: socket hang up
    at UIA2Proxy.command (C:\Users\ASUS\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\jsonwp-proxy\proxy.js:274:13)
    at runMicrotasks (&lt;anonymous&gt;)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at AndroidUiautomator2Driver.commands.getDevicePixelRatio (C:\Users\ASUS\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\node_modules\appium-uiautomator2-driver\lib\commands\viewport.js:14:10)
    at AndroidUiautomator2Driver.fillDeviceDetails (C:\Users\ASUS\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\node_modules\appium-uiautomator2-driver\lib\driver.js:244:28)
    at AndroidUiautomator2Driver.createSession (C:\Users\ASUS\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\node_modules\appium-uiautomator2-driver\lib\driver.js:230:7)
    at AppiumDriver.createSession (C:\Users\ASUS\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\lib\appium.js:387:35)</status>
</test>
<status status="FAIL" starttime="20240109 23:49:16.994" endtime="20240109 23:49:31.638"/>
</suite>
<suite id="s1-s1-s2" name="loginSuite" source="D:\Sanbercode\Sanbercode-Quiz-4-Bootcamp-QA-batch-52\suites\loginSuite.robot">
<test id="s1-s1-s2-t1" name="User should be able to login with valid credentials" line="15">
<kw name="Run Keywords" library="BuiltIn" type="SETUP">
<arg>Open Flight Application</arg>
<arg>AND Login with Valid Credentials</arg>
<doc>Executes all the given keywords in a sequence.</doc>
<kw name="Open Flight Application" library="base">
<kw name="Open Application" library="AppiumLibrary">
<arg>remote_url=${REMOTE_URL}</arg>
<arg>platformName=${PLATFORM_NAME}</arg>
<arg>platformVersion=${PLATFORM_VERSION}</arg>
<arg>deviceName=${DEVICE_NAME}</arg>
<arg>appPackage=${APP_PACKAGE}</arg>
<arg>appActivity=${APP_ACTIVITY}</arg>
<doc>Opens a new application to given Appium server.
Capabilities of appium server, Android and iOS,
Please check https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/server-args.md
| *Option*            | *Man.* | *Description*     |
| remote_url          | Yes    | Appium server url |
| alias               | no     | alias             |
| strict_ssl          | No     | allows you to send commands to an invalid certificate host like a self-signed one. |</doc>
<kw name="Capture Page Screenshot" library="AppiumLibrary">
<doc>Takes a screenshot of the current page and embeds it into the log.</doc>
<msg timestamp="20240109 23:49:45.404" level="FAIL">No application is open</msg>
<status status="FAIL" starttime="20240109 23:49:45.403" endtime="20240109 23:49:45.407"/>
</kw>
<msg timestamp="20240109 23:49:45.409" level="WARN">Keyword 'Capture Page Screenshot' could not be run on failure: No application is open</msg>
<msg timestamp="20240109 23:49:45.411" level="FAIL">WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to the remote server. Original error: socket hang up
Stacktrace:
UnknownError: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to the remote server. Original error: socket hang up
    at UIA2Proxy.command (C:\Users\ASUS\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\jsonwp-proxy\proxy.js:274:13)
    at runMicrotasks (&lt;anonymous&gt;)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at AndroidUiautomator2Driver.commands.getDevicePixelRatio (C:\Users\ASUS\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\node_modules\appium-uiautomator2-driver\lib\commands\viewport.js:14:10)
    at AndroidUiautomator2Driver.fillDeviceDetails (C:\Users\ASUS\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\node_modules\appium-uiautomator2-driver\lib\driver.js:244:28)
    at AndroidUiautomator2Driver.createSession (C:\Users\ASUS\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\node_modules\appium-uiautomator2-driver\lib\driver.js:230:7)
    at AppiumDriver.createSession (C:\Users\ASUS\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\lib\appium.js:387:35)</msg>
<status status="FAIL" starttime="20240109 23:49:31.665" endtime="20240109 23:49:45.413"/>
</kw>
<status status="FAIL" starttime="20240109 23:49:31.665" endtime="20240109 23:49:45.413"/>
</kw>
<status status="FAIL" starttime="20240109 23:49:31.664" endtime="20240109 23:49:45.413"/>
</kw>
<status status="FAIL" starttime="20240109 23:49:31.662" endtime="20240109 23:49:45.414">Setup failed:
WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to the remote server. Original error: socket hang up
Stacktrace:
UnknownError: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to the remote server. Original error: socket hang up
    at UIA2Proxy.command (C:\Users\ASUS\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\jsonwp-proxy\proxy.js:274:13)
    at runMicrotasks (&lt;anonymous&gt;)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at AndroidUiautomator2Driver.commands.getDevicePixelRatio (C:\Users\ASUS\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\node_modules\appium-uiautomator2-driver\lib\commands\viewport.js:14:10)
    at AndroidUiautomator2Driver.fillDeviceDetails (C:\Users\ASUS\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\node_modules\appium-uiautomator2-driver\lib\driver.js:244:28)
    at AndroidUiautomator2Driver.createSession (C:\Users\ASUS\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\node_modules\appium-uiautomator2-driver\lib\driver.js:230:7)
    at AppiumDriver.createSession (C:\Users\ASUS\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\lib\appium.js:387:35)</status>
</test>
<status status="FAIL" starttime="20240109 23:49:31.641" endtime="20240109 23:49:45.418"/>
</suite>
<status status="FAIL" starttime="20240109 23:49:16.988" endtime="20240109 23:49:45.421"/>
</suite>
<status status="FAIL" starttime="20240109 23:49:16.940" endtime="20240109 23:49:45.424"/>
</suite>
<statistics>
<total>
<stat pass="0" fail="2" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="0" fail="2" skip="0" id="s1" name="Sanbercode-Quiz-4-Bootcamp-QA-batch-52">Sanbercode-Quiz-4-Bootcamp-QA-batch-52</stat>
<stat pass="0" fail="2" skip="0" id="s1-s1" name="Suites">Sanbercode-Quiz-4-Bootcamp-QA-batch-52.Suites</stat>
<stat pass="0" fail="1" skip="0" id="s1-s1-s1" name="bookSuite">Sanbercode-Quiz-4-Bootcamp-QA-batch-52.Suites.bookSuite</stat>
<stat pass="0" fail="1" skip="0" id="s1-s1-s2" name="loginSuite">Sanbercode-Quiz-4-Bootcamp-QA-batch-52.Suites.loginSuite</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20240109 23:49:17.598" level="ERROR">Error in file 'D:\Sanbercode\Sanbercode-Quiz-4-Bootcamp-QA-batch-52\suites\loginSuite.robot' on line 4: Setting 'Test Setup' is not allowed in resource file.</msg>
<msg timestamp="20240109 23:49:17.600" level="ERROR">Error in file 'D:\Sanbercode\Sanbercode-Quiz-4-Bootcamp-QA-batch-52\suites\bookSuite.robot' on line 6: Error in file 'D:\Sanbercode\Sanbercode-Quiz-4-Bootcamp-QA-batch-52\suites\loginSuite.robot' on line 14: Resource file with 'Test Cases' section is invalid.</msg>
<msg timestamp="20240109 23:49:31.629" level="WARN">Keyword 'Capture Page Screenshot' could not be run on failure: No application is open</msg>
<msg timestamp="20240109 23:49:45.409" level="WARN">Keyword 'Capture Page Screenshot' could not be run on failure: No application is open</msg>
</errors>
</robot>
